packages/vite-plugin-cloudflare/src/runner/module-runner.ts

```diff
@@ -5,11 +5,24 @@ import type { WrapperEnv } from './env';
 
 let moduleRunner: ModuleRunner;
 
-export function createModuleRunner(env: WrapperEnv, webSocket: WebSocket) {
+// node modules using process.env don't find process in the global scope for some reason...
+// so let's define it here ¯\_(ツ)_/¯
+(globalThis as Record<string, unknown>).process = { env: {} };
```

Comment:
Not sure if this is the right thing to do here. Should we only provide `process.env` if `node_compat` is enabled or do you think it's still needed otherwise?

Reply:
yeah, I'd personally just keep it here for now and then wait for @petebacondarwin's nodejs changes and potentially make this like obsolete from those and remove it then

But I am also totally fine removing this for now and skip the tests that fail because of the missing `process`

Just let me know your preference 🙂👍

Reply:
What do you think @petebacondarwin? Do you want to keep it for now and then replace it later?

Reply:
Leave it in for now. It could be some time before I get this nodejs stuff working properly... 😢

Reply:
What would be super nice is to (create and?) link a bug for stuff like that

Reply:
done 👍